src/api/spec/controllers/search_controller_spec.rb

```diff
@@ -93,6 +93,20 @@
     end
   end
 
+  describe 'search for requests' do
+    let(:group) { create(:group) }
+    let(:bs_request) { create(:set_bugowner_request, state: :review, status: :review) }
+    let!(:review) { create(:review, by_group: group.title, bs_request: bs_request) }
+
+    before do
+      get :bs_request, params: { match: "state/@name='review'" }, format: :xml
```

Comment:
Isn't this matcher `status` now?

Reply:
We don't want to change the matcher, as that would mean changing the API. Our goal is to rename a field in the database without modifying the API.

Instead, we keep the matcher as is and change the field it compares against. See: https://github.com/openSUSE/open-build-service/pull/18173/files#diff-d6a9e4d72b743662ce44905f22e436cb67b28b0e94b66bbccc0470ec61b98809R201

Just in case it’s hard to see the changes on GitHub, here’s the relevant diff:

```diff
- 'state/@name' => { cpart: 'bs_requests.state' },
+ 'state/@name' => { cpart: 'bs_requests.status', function: "BsRequest.statuses['$VALUE']" },
```